index.js

```diff
@@ -164,6 +164,9 @@ function SpotifyResolve(createOpts) {
       if (error) {
         done(error);
       }
+      else if (response.statusCode === 401) {
+        done(new Error(response.statusCode), response);
```

Comment:
Since this a module for use in a lot of different kinds of applications, I think the `message` actually has to be a message. But we can enable that kind of simple checking via [the name property of the Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name)!